deps/chakrashim/src/v8message.cc

```diff
@@ -28,8 +28,18 @@ Local<String> Message::Get() const {
 }
 
 MaybeLocal<String> Message::GetSourceLine(Local<Context> context) const {
-  // CHAKRA-TODO: Figure out how to transmit this info...?
-  return Local<String>();
+  JsPropertyIdRef source = jsrt::IsolateShim::GetCurrent()
+    ->GetCachedPropertyIdRef(jsrt::CachedPropertyIdRef::source);
+  if (source == JS_INVALID_REFERENCE) {
```

Comment:
In what cases can this happen?  If it's only coding errors it seems better to just fast fail using CHAKRA_VERIFY or similar.  That said, I see a lot of examples where the result is just trusted and passed on, so maybe the check is not necessary?